### PR TITLE
chore(Wotan): Install and configure Wotan

### PR DESCRIPTION
**Pull request by ElijahKotyluk**
# Add Wotan
- [x] tslint.json
- [x] tsconfig.json
- [x] install @fimbul/wotan
- [x] install @fimbul/valtyr 
- [x] install tslint-consistent-codestyle(Removed)
- [x] .fimbullinter.yaml
- [x] .wotanrc.yaml
- [x] pkg scripts 
- [x] lerna scripts

Lerna for linting must have '--no-bail' included in the command prefixing run in order for Lerna to consistently lint every package without bailing on 'error 2'.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
   "scripts": {
     "commit": "npx git-cz",
     "test": "lerna run test --stream",
-    "lint": "lerna run lint --stream",
-    "lint:fix": "lerna run lint:fix --stream",
+    "lint": "lerna run --no-bail lint:wotan --stream",
+    "lint:fix": "lerna run --no-bail lint:wotan:fix --stream",
```
Comment (dsseng) [inline package.json]: Tests aren't fixable, please make them fixable

File: package.json [hunk 2]
```diff
@@ -12,8 +12,10 @@
   "scripts": {
     "commit": "npx git-cz",
     "test": "lerna run test --stream",
-    "lint": "lerna run lint --stream",
-    "lint:fix": "lerna run lint:fix --stream",
+    "lint": "lerna run --no-bail lint:wotan --stream",
```
Comment (dsseng) [inline package.json:15]: lint should lint everything, lint:fix should fix everything

File: packages/matchers/package.json [hunk 3]
```diff
@@ -7,8 +7,10 @@
   "author": "Elijah Kotyluk <elijah@elijahkotyluk.com>",
   "license": "MIT",
   "scripts": {
```
Comment (dsseng) [inline packages/matchers/package.json:9]: same thing with scripts

File: packages/mock/package.json [hunk 4]
```diff
@@ -7,8 +7,10 @@
   "author": "Elijah Kotyluk <elijah@elijahkotyluk.com>",
   "license": "MIT",
   "scripts": {
```
Comment (dsseng) [inline packages/mock/package.json:9]: same thing with scripts

File: packages/mock/tsconfig.json [hunk 5]
```diff
@@ -14,7 +16,8 @@
       "es2018",
       "es2019",
       "es2020",
-      "esnext"
+      "esnext",
+      "dom"
```
Comment (dsseng) [inline packages/mock/tsconfig.json]: do we need `dom` here?

Comment (ElijahKotyluk) [reply packages/mock/tsconfig.json]: Removed

File: package.json [hunk 6]
```diff
@@ -12,8 +12,8 @@
...
```
Comment (dsseng) [inline package.json:15]: probably bail is good here imho

Comment (ElijahKotyluk) [reply package.json:15]: With bail Lerna won't allow consecutive lint calls on a consistent basis for some reason. I was playing around with it earlier and --no-bail was the only way I could get it to work 100% of the time. It kept bailing on error 2
edit: This is only an issue with lint command, both tslint and wotan had this same issue.

**Review by dsseng — APPROVED** [review]

**Pull request merged**